master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -256,11 +304,11 @@ The GitLab hook is as simple as GitHub one and it also takes no options.
 ::
 
     c['www'] = dict(...,
-        change_hook_dialects={ 'gitlab' : True }
+        change_hook_dialects={'gitlab': True}
     )
 
 When this is setup you should add a `POST` service pointing to ``/change_hook/gitlab`` relative to the root of the web status.
-For example, it the grid URL is ``http://builds.example.com/bbot/grid``, then point GitLab to ``http://builds.example.com/change_hook/gitlab``.
+For example, if the grid URL is ``http://builds.example.com/bbot/grid``, then point GitLab to ``http://builds.example.com/change_hook/gitlab``.
```

Comment:
There is no grid url anymore in buildbot nine, so you would say:

    For example, if the home page URL is ``http://builds.example.com/bbot/#/``, then point GitLab to ``http://builds.example.com/bbot/change_hook/gitlab``.